platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/activity/BaseActivityTest.java

```diff
@@ -72,7 +73,7 @@ protected void waitLoop() {
   }
 
   protected void waitLoop(long waitTime) {
```

Comment:
Small nitpick, can we rename `#waitLoop()` to match with the new `WaitAction` class?